### DISCUSSION TITLE
Attribute card

### DISCUSSION THREAD

**Discussion by RayMathew** [Buzzpy/Dev-Encyclopedia#90] [Q&A] [answered]
The card for Attribute defines it as an HTML attribute, but it is a generic term that can mean other things too.

> In computer programming, attributes are characteristics or properties that can be assigned to elements like classes, objects, or variables. They can provide additional information or behavior to these elements, helping to define their specific traits. Attributes can also be used to manipulate or reference elements, fields, or files.

**Comment by Buzzpy** [comment, marked as answer]
I see. We can update the "Attribute" card with the above info and also add a new card called "HTML Attributes" with existing info, right?

**Reply by RayMathew** [reply]
Yep. I can do that.

**Reply by RayMathew** [reply]
This is complete https://github.com/Buzzpy/Dev-Encyclopedia/pull/99
